ui/segment.reel/segment.css

```diff
@@ -0,0 +1,12 @@
+.montage-Segment {
+    /* to vertical align text labels */
+    display: -webkit-box;
+    display: -webkit-flex;
+    display: -ms-flexbox;
+    display: flex;
+
+    -webkit-box-align: center;
+    -webkit-align-items: center;
+    -ms-flex-align: center;
+    align-items: center;
+}
```

Comment:
Prefixes are chosen by using https://github.com/postcss/autoprefixer, with default browser choise of `> 1%, last 2 versions, Firefox ESR, Opera 12.1`, more details here: https://github.com/ai/browserslist